shell/src/main/java/alluxio/cli/fs/command/LoadCommand.java

```diff
@@ -144,8 +147,11 @@ private void runLoadTask(AlluxioURI filePath, URIStatus status, boolean local)
       }
       Protocol.OpenUfsBlockOptions openUfsBlockOptions =
           new InStreamOptions(status, options, conf).getOpenUfsBlockOptions(blockId);
-      cacheBlock(blockId, dataSource, status, openUfsBlockOptions);
+      if (!cacheBlock(blockId, dataSource, status, openUfsBlockOptions)) {
+        return false;
+      }
```

Comment:
Will this prevent the other blocks from loading, if one of the blocks fails to be cached?

Reply:
yeah, do you think this make sense?